<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClonaLiSAForm</class>
 <widget class="QWidget" name="ClonaLiSAForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClonaLiSA</string>
  </property>
  <layout class="QHBoxLayout" name="topHLayout" stretch="0">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="leftBar">
      <layout class="QVBoxLayout" name="leftVBox" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutInput">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnConfig">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Config</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelInput">
           <property name="text">
            <string>Input directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inp_edit"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowseInput">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutModel">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelModel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Model:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="model_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowseModel">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutThresholds">
         <item>
          <widget class="QLabel" name="labelFlowThr">
           <property name="text">
            <string>Flow thr:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="flow_edit"/>
         </item>
         <item>
          <widget class="QLabel" name="labelMaskThr">
           <property name="text">
            <string>Mask thr:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mask_edit"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutFilter">
           <item>
            <widget class="QLabel" name="labelFilter">
             <property name="text">
              <string>Img Filter:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filt_edit">
             <property name="text">
              <string>bright</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layoutZ">
             <item>
              <widget class="QLabel" name="labelZ">
               <property name="text">
                <string>Channel Order:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="z_edit">
               <property name="text">
                <string>0,1,2</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnRunOmnipose">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Run Omnipose</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="cb_outlines">
             <property name="text">
              <string>Save outlines</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_cellprob">
             <property name="text">
              <string>Save cell prob</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_flows">
             <property name="text">
              <string>Save flows</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="progressSubdir">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressOverall">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutPlate">
         <item>
          <widget class="QLabel" name="labelPlate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Plate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="plate_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutGroup">
         <item>
          <widget class="QPushButton" name="btnNewGroup">
           <property name="text">
            <string>New Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="view_combo">
           <item>
            <property name="text">
             <string>Imaged Wells</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="value_edit">
           <property name="placeholderText">
            <string>Value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnApplyGroup">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveGroups">
           <property name="text">
            <string>Save Groups</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>8</number>
         </property>
         <property name="columnCount">
          <number>12</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutCSV">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelCSV">
           <property name="text">
            <string>all_data_csv:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="csv_edit"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowseCSV">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutFE1">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelFE1">
           <property name="text">
            <string>Fixed Effect 1:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fe1_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutFE2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelFE2">
           <property name="text">
            <string>Fixed Effect 2:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fe2_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutInteraction">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="cb_interaction">
           <property name="text">
            <string>Include Interaction</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelRefLevel">
           <property name="text">
            <string>Reference Level:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ref_level_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnRunR">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Run Growth Rate Analysis in R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Output:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="log">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rightPanel">
      <layout class="QVBoxLayout" name="rightVBox" stretch="0">
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
